api/src/classes/pagination.py

```diff
@@ -36,13 +41,35 @@ class CompanyPagination(Pagination):
 
     def __init__(
         self,
-        page: int = PaginationDefaults.PAGE.value,
-        limit: int = PaginationDefaults.LIMIT.value,
+        page: int = Check(default=PaginationDefaults.PAGE.value, ge=1),
+        limit: int = Check(default=PaginationDefaults.LIMIT.value, ge=1),
+        *,
+        ascending: bool = False,
+    ) -> None:
+        super().__init__(page=page, limit=limit)
+        self.ascending: bool | None = ascending
```

Comment:
`ascending` should never be None, even if the user doesn't provide a value it's already False by default